src/rgw/rgw_pubsub.h

```diff
@@ -342,12 +342,14 @@ struct rgw_pubsub_dest {
   std::string arn_topic;
   bool stored_secret = false;
   bool persistent = false;
+  // rados object name of the persistent queue in the 'notif' pool
+  std::string persistent_queue;
```

Comment:
i do not see a reason on why we need to additional variable to struct, since we are able to generate the name based on tenant n topic name
`get_account_or_tenant(s->owner.id), ":", topic_name)`

Reply:
this is how we handle backward compatibility with queues that don't have a namespace. this string will contain the topic name only